extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/logging/RequestLoggerRecorder.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.vertx.http.runtime.logging;
+
+import io.quarkus.runtime.annotations.Recorder;
+import io.vertx.core.Handler;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.handler.LoggerHandler;
+
+@Recorder
+public class RequestLoggerRecorder {
+
+    private static final RequestLogMessageFormatter REQUEST_LOG_MESSAGE_FORMATTER = new RequestLogMessageFormatter();
```

Comment:
No need to be static since it will only be used when the `LONG` format is chosen